### ISSUE TITLE
can't run on Big sur 11.2.1

### ISSUE THREAD

**Issue by abay2204** [mxschmitt/playwright-go#106]
Hi 
i am just update my Mac OS to last version today
but i can't run on my program

this the problem is

```
2021/02/16 20:48:35 Error Browser%!(EXTRA *fmt.wrapError=could not send message: could not send message to server: No executable path is specified. Pass "executablePath" option directly.
Note: use DEBUG=pw:api environment variable and rerun to capture Playwright logs.)
exit status 1
```

```
browser, err := pw.Chromium.Launch(playwright.BrowserTypeLaunchOptions{
			Headless: playwright.Bool(false),
		})

```
i had change the browser chromium, firefox or webkit but nobody can work of them

thanks before

**Comment by kennylixi** [comment]
@abay2204 Have you solved the problem? I have the same problem

**Comment by kennylixi** [comment]
(node:80845) UnhandledPromiseRejectionWarning: Error: ERROR: Playwright does not support chromium on mac11.2
    at Object.assert (/snapshot/playwright-cli/node_modules/playwright/lib/utils/utils.js:58:15)
    at revisionURL (/snapshot/playwright-cli/node_modules/playwright/lib/install/browserFetcher.js:143:13)
    at Object.downloadBrowserWithProgressBar (/snapshot/playwright-cli/node_modules/playwright/lib/install/browserFetcher.js:168:17)
    at async validateCache (/snapshot/playwright-cli/node_modules/playwright/lib/install/installer.js:103:9)
    at async Object.installBrowsersWithProgressBar (/snapshot/playwright-cli/node_modules/playwright/lib/install/installer.js:60:5)
(node:80845) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:80845) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by mxschmitt** [comment]
I have to upgrade it to the latest Playwright version, this requires changes in the generation scripts, hope to be able to do that in the next few days/weeks.

**Comment by mxschmitt** [comment]
Should be fixed in v0.192
